src/releasers/node.ts

```diff
@@ -27,25 +32,17 @@ import {PackageJson} from '../updaters/package-json';
 import {SamplesPackageJson} from '../updaters/samples-package-json';
 
 export class Node extends ReleasePR {
-  async getOpenPROptions(
+  private pkgJsonContents?: GitHubFileContents;
```

Comment:
Is this just an optimization so that we only load once?

Reply:
yeah - the original `_run` code loads the contents at the beginning only once and passes it to the `PackageJson` Updater as well as sets `self.packageName` - but now that I've tried to make `getPackageName()` available potentially prior to any "run" I thought I'd cache it here. But maybe potential caching bugs aren't worth the small cost of an extra API call or two?